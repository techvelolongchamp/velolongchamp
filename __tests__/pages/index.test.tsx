import React from 'react'
import { RenderWithProvider as render } from '../provider'

import Home from '../../pages/index'

const useRouter = jest.spyOn(require('next/router'), 'useRouter')

jest.mock('next/image', () => ({
  __esModule: true,
  default: (props: any) => {
    const keys = Object.keys(props)
    const simplifiedProps = {}
    keys.forEach((key) => {
      simplifiedProps[key.toLowerCase()] = props[key]
    })
    // eslint-disable-next-line @next/next/no-img-element, jsx-a11y/alt-text
    return <img {...simplifiedProps} />
  },
}))

describe('Home unit tests', () => {
  it('should render correctly', () => {
    window.matchMedia = jest.fn().mockImplementation(() => ({
      addListener: jest.fn(),
      removeListener: jest.fn(),
      matches: false,
    }))
    useRouter.mockImplementation(() => ({ locale: 'fr', pathname: '/' }))
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
})
