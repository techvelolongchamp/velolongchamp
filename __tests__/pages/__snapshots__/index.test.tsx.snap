// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home unit tests should render correctly 1`] = `
.c22 {
  background: #f7f7F7;
  border-top: 1px solid #f7f7F7;
  color: #999999;
  padding: 10px 0 5px;
  text-align: center;
}

.c22 a {
  color: #999999;
}

.c22 a:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c23 {
  background: #0d171b;
  bottom: 0;
  -webkit-transition: all 1s ease;
  transition: all 1s ease;
  color: white;
  font-size: 14px;
  padding: 10px;
  position: fixed;
  text-align: center;
  width: 100vw;
  z-index: 1000;
}

.c23 div {
  margin-top: 10px;
}

.c23 a {
  background: #6d6e71;
  border-radius: 5px;
  color: #fff;
  margin: 0 10px;
  padding: 5px;
}

.c23 button {
  background: #419a1c;
  border-radius: 5px;
  color: #fff;
  margin: 0 10px;
  outline: done;
  padding: 5px;
  width: 40px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
}

.c4 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  margin-top: 80px;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: darkorange;
  border-bottom-left-radius: 30px;
  border-top-left-radius: 30px;
  color: white;
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 40px;
  font-weight: 100;
  position: fixed;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  right: 200px;
  top: 120px;
  z-index: 10;
}

.c1 svg {
  margin: 10px;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: darkorange;
  color: #fff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: fixed;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  right: 0;
  top: 120px;
  z-index: 10;
  width: 200px;
  font-size: 16px;
  padding: 10px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  text-align: center;
  border-bottom-left-radius: 10px;
}

.c3 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #fff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 16px;
  height: 30px;
  margin: 10px 20px 0;
  padding: 5px;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  background: #419a1c;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 120px;
  border-radius: 5px;
}

.c3:hover {
  color: #f7f7F7;
}

.c14 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  font-size: 12px;
  margin: 0 5px;
}

.c15 {
  background: none;
  border: none;
  outline: none;
  color: black;
  padding: 5px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c15:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c10 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  list-style-type: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 16px;
  height: 30px;
  margin: 0 20px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c11 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #999999;
  cursor: pointer;
  padding: 0 20px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c11:hover {
  color: #666666;
}

.c5 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  z-index: 15;
  background: #f7f7F7;
  padding: 5px 10px;
  border-bottom: 1px solid #fff;
  color: #0d171b;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c12 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 0 15px;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
}

.c13 {
  background: #6d6e71;
  border-radius: 5px;
  color: #fff;
  cursor: pointer;
  font-size: 16px;
  padding: 10px;
}

.c9 {
  height: 70px;
  padding: 5px 0;
}

.c16 {
  min-height: 600px;
  height: calc(100vh - 80px);
}

.c18 {
  margin: 0 auto;
  max-width: 750px;
  padding: 10px;
  height: inherit;
}

.c17 {
  min-height: 600px;
  background: #f7f7F7;
}

.c19 {
  color: #419a1c;
  font-size: 30px;
  margin: 20px 0;
}

.c20 {
  padding-bottom: 30px;
}

.c20 h3 {
  font-size: 16px;
  margin: 20px 0;
  color: #9e9577;
  font-weight: 600;
}

.c20 p {
  font-size: 14px;
  line-height: 20px;
  color: #0d171b;
  padding: 10px 0;
  text-align: justify;
}

.c20 ol {
  list-style: none;
  counter-reset: styled-counter;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin: 0;
  padding: 0;
}

.c20 ol li {
  counter-increment: styled-counter;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px 0;
  width: 50%;
}

.c20 ol li h4 {
  color: #6d6e71;
  font-size: 16px;
  font-weight: 500;
  padding-right: 10px;
}

.c20 ol li::before {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #6d6e71;
  content: '0' counter(styled-counter);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-family: 'Abril Fatface',serif;
  font-size: 30px;
  font-weight: bold;
  line-height: 1;
  margin-right: 5px;
}

.c20 a {
  color: #6d6e71;
  font-weight: 800;
}

.c20 ul {
  list-style: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin: 0;
  padding: 0;
}

.c20 ul li {
  counter-increment: styled-counter;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 10px 0;
  width: 50%;
}

.c20 ul li h4 {
  color: #6d6e71;
  font-size: 16px;
  font-weight: 500;
  padding-right: 10px;
}

.c21 {
  text-align: center;
}

.c21 iframe {
  width: 560px;
  height: 315px;
}

@media (max-width:640px) {
  .c4 {
    margin-top: 110px;
  }
}

@media screen and (max-width:640px) {
  .c10 {
    font-size: 14px;
    padding: 0;
    margin: 0 10px;
  }
}

@media screen and (max-width:640px) {
  .c11 {
    padding: 0 10px;
  }
}

@media screen and (max-width:640px) {
  .c13 {
    font-size: 14px;
  }
}

@media (max-width:640px) {
  .c16 {
    height: calc(100vh - 110px);
  }
}

@media (max-width:640px) {
  .c20 ol li {
    width: 100%;
  }
}

@media (max-width:640px) {
  .c20 ul li {
    width: 100%;
  }
}

@media (max-width:640px) {
  .c21 iframe {
    width: 100%;
    max-width: 560px;
    max-height: 315px;
    height: 100%;
    height: -moz-available;
    height: -webkit-fill-available;
    height: -webkit-fill-available;
    height: -moz-available;
    height: fill-available;
  }
}

<div>
  <div
    class="c0"
  >
    <div>
      <div
        class="c1"
        data-testid="show-alert"
        id="show-alert"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10.29 3.86L1.82 18a2 2 0 0 0 1.71 3h16.94a2 2 0 0 0 1.71-3L13.71 3.86a2 2 0 0 0-3.42 0z"
          />
          <line
            x1="12"
            x2="12"
            y1="9"
            y2="13"
          />
          <line
            x1="12"
            x2="12.01"
            y1="17"
            y2="17"
          />
        </svg>
      </div>
      <div
        class="c2"
      >
        Signaler un accident ou probl√®me sur l'anneau
        <a
          class="c3"
          data-testid="notice-alert"
          href="https://docs.google.com/forms/d/e/1FAIpQLSdmrvUuZ953-3P8NVmNjiG8N78hY1lcdg_hVbr8lnYEqSjM9A/viewform?vc=0&c=0&w=1"
          id="notice-alert"
          rel="noopener noreferrer"
          target="_blank"
          title="Formulaire"
        >
          Formulaire
        </a>
      </div>
    </div>
    <main
      class="c4"
    >
      <header
        class="c5"
        id="header"
      >
        <div
          class="c6"
        >
          <div
            class="c7"
          >
            <a
              class="c8"
              data-testid="title-link"
              href="/"
              id="logo-link"
            >
              <img
                alt="logo"
                class="c9"
                src="/logo.png"
              />
            </a>
            <nav>
              <ul
                class="c10"
                data-testid="navbar"
              >
                <li>
                  <a
                    class="c11"
                    href="/"
                    id="nav-link-home"
                  >
                    A propos
                  </a>
                </li>
                <li>
                  <a
                    class="c11"
                    href="/blog"
                    id="nav-link-blog"
                  >
                    Actualit√©s
                  </a>
                </li>
                <li>
                  <a
                    class="c11"
                    href="/calendar"
                    id="nav-link-calendar"
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M0 464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V192H0v272zm320-196c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12h-40c-6.6 0-12-5.4-12-12v-40zm0 128c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12h-40c-6.6 0-12-5.4-12-12v-40zM192 268c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12h-40c-6.6 0-12-5.4-12-12v-40zm0 128c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12h-40c-6.6 0-12-5.4-12-12v-40zM64 268c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12H76c-6.6 0-12-5.4-12-12v-40zm0 128c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12H76c-6.6 0-12-5.4-12-12v-40zM400 64h-48V16c0-8.8-7.2-16-16-16h-32c-8.8 0-16 7.2-16 16v48H160V16c0-8.8-7.2-16-16-16h-32c-8.8 0-16 7.2-16 16v48H48C21.5 64 0 85.5 0 112v48h448v-48c0-26.5-21.5-48-48-48z"
                      />
                    </svg>
                  </a>
                </li>
              </ul>
            </nav>
          </div>
          <div
            class="c12"
          >
            <a
              class="c13"
              href="/join"
              id="join-link"
            >
              Adh√©rer gratuitement
            </a>
            <div
              class="c14"
            >
              <button
                class="c15"
                data-testid="select-en"
                href="/"
                id="select-en"
              >
                EN
              </button>
            </div>
          </div>
        </div>
      </header>
      <section
        class="c16"
      >
        <img
          alt="Banniere.JPG"
          blurdataurl="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/4gIoSUNDX1BST0ZJTEUAAQEAAAIYAAAAAAQwAABtbnRyUkdCIFhZWiAAAAAAAAAAAAAAAABhY3NwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAA9tYAAQAAAADTLQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAlkZXNjAAAA8AAAAHRyWFlaAAABZAAAABRnWFlaAAABeAAAABRiWFlaAAABjAAAABRyVFJDAAABoAAAAChnVFJDAAABoAAAAChiVFJDAAABoAAAACh3dHB0AAAByAAAABRjcHJ0AAAB3AAAADxtbHVjAAAAAAAAAAEAAAAMZW5VUwAAAFgAAAAcAHMAUgBHAEIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFhZWiAAAAAAAABvogAAOPUAAAOQWFlaIAAAAAAAAGKZAAC3hQAAGNpYWVogAAAAAAAAJKAAAA+EAAC2z3BhcmEAAAAAAAQAAAACZmYAAPKnAAANWQAAE9AAAApbAAAAAAAAAABYWVogAAAAAAAA9tYAAQAAAADTLW1sdWMAAAAAAAAAAQAAAAxlblVTAAAAIAAAABwARwBvAG8AZwBsAGUAIABJAG4AYwAuACAAMgAwADEANv/bAEMAFA4PEg8NFBIQEhcVFBgeMiEeHBwePSwuJDJJQExLR0BGRVBac2JQVW1WRUZkiGVtd3uBgoFOYI2XjH2Wc36BfP/bAEMBFRcXHhoeOyEhO3xTRlN8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fP/AABEIAZkCZgMBIgACEQEDEQH/xAAYAAEBAQEBAAAAAAAAAAAAAAAAAQQDBf/EABQQAQAAAAAAAAAAAAAAAAAAAAD/xAAXAQEBAQEAAAAAAAAAAAAAAAAAAQID/8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEQMRAD8A8cBzUAAQAABABFAAQAQAARUQAAQABFQAAEAUQAEABEVFQRUARUUQBRAAQEFEVAEVAEVBQABAAAFAAAAAAUAQVAFAUABAABUUAAAAHoAMKAAAAAAACCKiKACAACKiAACAAIqAAgAIoAAiKgCKioIqAIqKIAoiKgCKgoioAioAioKAAgAoAAAAAAACgAACKIoACoAAKgCgAAA9ABhQAAAQEAVAAARQAQEAAAEVEAEAAAQAEVFBFQBFQBAVBABAFERUUEVAEVBUAAQAEAUQAAAABQRQAAAAFRRAABUAUBQAEAAAAAAegAwAAAAAAAAAIgAAAACAACAgAAgAIAAogAIAogIIAigioAgKCACACoCAAgAAqAAAgKgCgAAAKIoAAKICKAAqCigCAAAAAAPQEGBUAAAAAAAABAEAAAAAEAAEAAAQAEABAAQFBAEEBRAQAEUEVBRFQBAAQAEAUEAAAABQQBQAAAURRAABUAUBQAEAAUQBRAHoCDAogCiAKIAAAAAAiCoAAAAgACAAAIAAIACAAigCAIqKCAIgCiAgoCAIqAIqAAgoAAgCgAAAAAKIAoAiiAKIoCoKKICKIAoAAAN4gyAAACAAAAACAogCoCAAAIAAAIAAIAACAigACAgAIoIqCCAoIIKAgAIACAAgoAAICqIAogCiAKAAqAiiKAACiAKAqAAAAAANwDIAAAAAIAAAgCiAAAAIgogACAqAACAAiioICoIACAAigCCCAqiACAgAIACAAgoAAIAogKogCiKAAAqAKAIKgCiAKAIAKAAAANwgyKIAogCiAKIIKIAqAACAogCoAAgCoAAgACAAgAIACKAIACCCAKIIoAgAIACAAgoAoCAKICqIAoigAAKgiKIoCoAogCgCACgqAKIA2gMgAAAAAAAAAgCAKIAqAACAogACAqAAIAAgAIoAgAIAioAgKCCAAgCAAgKCAKAgKIAogCiAKIooAIKgCiKAACiAKIIigKAAAANogyiiAKIAogKogCiAKIIAAAICiAKgACAAACAACKAIAIACACAAgigCAIACCKKgAIAACqAAAAAAAAAIKIAoigAAKgIogCiAKIAoANgDKAAAAAICiAKICqIAogCiAKIAAgKIAAgKggKgAIACAAggKgigIAIAggiqAgAIoqAAAAqAKIAogCiAKAAAKAAACACCiAKIAogCiANogyiiAKIAogCiAKIAogAAKAACAKIAAgKIACAAIACAAgAIoIAgggAIqgIACKAAAAAAAAAAAAAAAAAAAAAAAACoAogCiAKIA2CDDKiAqiAKIAogCiAKIAogCiAKICgICiAAICoACAoIACAAggioIAIKCAKCCgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADWIMsqIAogCiAKICqIAogCoAAAAICiAqoAAgACAAgAIIAgAgoIACAAgKoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADSAyyAAAAAAAAACgAAAAgCiAKgACAKgAIAAgAgKCAAggACggCgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANADLIAAAAAAAAAAAAAKCAKgAAgKIAAgAIoqCAqCAqCAAKCAKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA7gIwAAACgCAAAAAAAIAogAAACCgCggAAgAIACKAACAKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA7gIwAAAAAAIqCgAACAAAIKAAAIAAAgAAgoAogACKgoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADuAjAAAAKAAgAgAKAgAAAACAAAAgAAgAAogKAAqAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA7gIyAAAAAAgAAACKgAAAIAAAioAACACgIAAoIqCgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAO4CMgAAAAAIAAAAioAAAgCAAqAAAgAAoioAAqoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADuAjIAAAAAAioAAAioAAIgACKgoAAioAAKIqAAKqAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA0IojKCgIAAAAigIAAioAAIgACKgoAAioAAKIAACqgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANACMgACKgAAAAIAAACACIAAioKAAIqAACoAAAqoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADQAMgCAACAAAAgAAAIAIgACKiKAAIqKAAqAAAKqAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA/9k="
          layout="fill"
          objectfit="cover"
          placeholder="blur"
          src="test-file-stub"
        />
      </section>
      <section
        class="c17"
      >
        <div
          class="c18"
        >
          <h2
            class="c19"
          >
            Pr√©sentation
          </h2>
          <article
            class="c20"
          >
            <h3>
              Qui sommes-nous ?
            </h3>
            
    
            <p>
              VELO LONGCHAMP est une association de cyclistes pour les cyclistes
            </p>
            
    
            <p>
              A l‚Äôorigine, nous avons port√© un projet de r√©novation de l‚Äôanneau de Longchamp dans le cadre du budget participatif Paris 2020. Grand succ√®s puisque notre projet a √©t√© retenu et des fonds allou√©s (500 K‚Ç¨ provisionn√©s) ! Notre mission premi√®re est de suivre ce projet pour que la r√©novation de l‚Äôanneau soit rapide et r√©ponde aux besoins des cyclistes.
            </p>
            
    
            <p>
              Mais notre vision est plus large et pourrait se r√©sumer ainsi : 
              <strong>
                D√©fendre et promouvoir l‚Äôanneau de Longchamp pour que toutes et tous, nous puissions faire du v√©lo dans les meilleures conditions.
              </strong>
            </p>
            
    
            <p>
              Ainsi, nous nous impliquons activement dans diff√©rents projets comme le d√©veloppement de la pratique du v√©lo par les jeunes et le soutien de la pratique handisport. Rappelons qu‚Äôen √©tant prot√©g√© de la circulation des voitures, l‚Äôanneau de Lonchamp est particuli√®rement adapt√© √† l‚Äôentrainement des sportifs handisports.
            </p>
            
    
            <p>
              Fort de l‚Äôadh√©sion de plusieurs centaines de membres, et de l‚Äôaffiliation de nombreux clubs sportifs parisiens et franciliens, nous sommes fiers de compter dans nos rangs tous les profils et toutes les pratiques cyclistes, sans distinction d‚Äô√¢ge, de genre, de niveau, ou m√™me de type de v√©lo ! Nous avons par exemple des pratiquants de tandem (certains malvoyants ou aveugles) et de v√©lo couch√©.
            </p>
             
    
            <p
              id="statuts-links"
            >
              <a
                href="/statuts"
              >
                Consulter nos statuts
              </a>
            </p>
            <h3>
              Pourquoi une association V√©lo Longchamp ?
            </h3>
            
    
            <p>
              L‚Äôanneau cyclable de Lonchamp est l‚Äôun des rares et pr√©cieux endroits o√π l‚Äôon peut rouler √† son rythme, en s√©curit√©, prot√©g√© des voitures. D‚Äôailleurs, c‚Äôest le segment le plus roul√© de France, de tr√®s loin, d‚Äôapr√®s l‚Äôapplication Strava. C‚Äôest donc un site sportif incontournable !
            </p>
            
    
            <p>
              Comme l‚Äôanneau similaire de Vincennes (Polygone), l'anneau cyclable de Longchamp est devenu une v√©ritable institution pour les cyclistes sportifs de Paris et de sa proche banlieue. Des centaines de cyclistes sportifs amateurs (et professionnels) s'y entrainent tous les jours ainsi que de nombreux marcheurs et coureurs √† pied qui tournent en sens inverse sur son trottoir int√©rieur.
            </p>
            
    
            <p>
              Pourtant, les ann√©es passent et les conditions se d√©gradent : rev√™tement abim√©, marquage au sol effac√©, signalisation d√©faillante ou absente, conflits d'usage fr√©quents avec les v√©hicules stationnant ou entrant et sortant de l'hippodrome, manque de concertation avec les autres usagers. Il est temps de r√©am√©nager l‚Äôanneau pour que nous, et les g√©n√©rations suivantes, puissions rouler dans des conditions parfaitement s√ªres et confortables.
            </p>
            
    
            <h3>
              Decouvrez les r√®gles essentielles pour rouler √† Longchamp :
            </h3>
            <div
              class="c21"
            >
              <iframe
                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
                allowfullscreen=""
                frameborder="0"
                src="https://www.youtube.com/embed/ZTRj9R6s-JI"
                title="Les r√®gles essentielles de Longchamp"
              />
            </div>
          </article>
        </div>
      </section>
    </main>
    <footer
      class="c22"
    >
      <p>
        ¬©Copyright 2019
         - 2022
         - V√©lo Longchamp -
         
        Tous droits r√©serv√©s
      </p>
      <p>
        -
      </p>
      <p
        id="legal-link"
      >
        <a
          href="/legal"
        >
          Mentions l√©gales
        </a>
      </p>
    </footer>
    <div
      class="c23"
    >
      <p>
        En poursuivant votre navigation sur notre site, vous acceptez l'utilisation de cookies afin de nous permettre d'am√©liorer votre exp√©rience utilisateur.
      </p>
      <div>
        <button
          data-testid="accept-cookie"
          id="accept-cookie"
        >
          OK
        </button>
        <a
          href="/legal"
        >
          En savoir plus
        </a>
      </div>
    </div>
  </div>
</div>
`;
